### PR TITLE
[webpack] Precompile assets on test env

### PR DESCRIPTION
**Pull request by didierofrivia**
**What this PR does / why we need it**:
> Fixes possible stale assets when running the test battery

### CODE REVIEW

**Comment by hallelujah** [comment]
Don't we need that too?
https://github.com/3scale/porta/blob/6e8c916a8cbdd0dffcd74ea7fad699607cd5de9f/config/webpacker.yml#L42-L44

**Comment by hallelujah** [comment]
Hmm approved but if we are pre-compiling then better setting `compile: false`
At least it will behave like in production environment which is good as we will know if something went wrong

File: config/webpacker.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ development:
 
 test:
   <<: *default
-  compile: true
+  compile: false
```
Comment (hallelujah) [inline config/webpacker.yml]: So when developing you will need to pre-compile the assets everytime? Or does webpacker-dev-server do it for you? Would be good to add some instructions here or probably use some environment. to control this.

Comment (didierofrivia) [reply config/webpacker.yml]: on development the assets will be compiled when they are required. You could also use webpacker-dev-server for autoloading and other debugging goodness

Comment (hallelujah) [reply config/webpacker.yml]: But this state that they will not be compiled when you test locally

Comment (didierofrivia) [reply config/webpacker.yml]: not the tests, sadly, you'll need to precompile them manually.

Comment (hallelujah) [reply config/webpacker.yml]: That is a no go without instructions. But can't we have this with environment switch?

```
compile: <%=  ENV.fetch('WEBPACKER_COMPILE', true) %>
```

Then in the CI build set this environment to `false`

Comment (hallelujah) [reply config/webpacker.yml]: Answering: https://github.com/rails/webpacker/issues/956

So no :/

**Pull request merged**